plutus-tx-plugin/test/IntegerLiterals/NoStrict/NegativeLiterals/Spec.hs

```diff
@@ -0,0 +1,58 @@
+{-# LANGUAGE BangPatterns          #-}
+{-# LANGUAGE DataKinds             #-}
+{-# LANGUAGE FlexibleContexts      #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedStrings     #-}
+{-# LANGUAGE ScopedTypeVariables   #-}
+{-# LANGUAGE TemplateHaskell       #-}
+{-# LANGUAGE TypeApplications      #-}
+
+{-# LANGUAGE NegativeLiterals      #-}
+{-# LANGUAGE NoStrict              #-}
+
+{-# OPTIONS_GHC -fplugin PlutusTx.Plugin #-}
```

Comment:
worth a comment, since basically all the other tests run with simplification off

Reply:
Added:
> -- This module runs the PIR and UPLC simplifiers, because (1) we want to verify that
-- `integerNegate` is compiled away; (2) it is easier to tell from the optimized PIR
-- whether or not the signs of the numbers are correct, which is ultimately what we
-- care about.